.github/workflows/stale.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     - uses: actions/stale@v9
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        stale-issue-message: 'This issue has been open one year with no activity.  Consequently, it is being marked with the "stale" label.  What this means is that the issue will be automatically closed in 30 days unless more comments are added or the "stale" label is removed.  Comments that provide new information on the issue are especially welcome: is it still reproducible? did it appear in other contexts? how critical is it? etc.'
+        stale-issue-message: 'This issue has been open one year with no activity and without being identified as a bug. Consequently, it is being marked with the "stale" label to see if anyone has comments that provide new information on the issue. Is the issue still reproducible? Did it appear in other contexts? How critical is it? Did you miss this feature in a new setting? etc. Without new comments, the issue will *not* be closed and it will remain alive with the "stale" label for triaging reason.'
```

Comment:
The "Without new comments... for triaging reason" part could be omitted, since it doesn't call for any action from the reporter of the issue or those who follow it.

Reply:
The message needs some emphasis that the stale bot will not close the issue. Maybe simply

> Note that the issue will *not* be closed in the absence of new activity: the "stale" label is only used for triaging reason.